content-repo/extra-docs/releases/20.6.1.md

```diff
@@ -0,0 +1,595 @@
+
+---
+
+### CSV Feed Pack v1.0.1
+#### Integrations
+##### CSVFeed  
+Changed the docker image to support auto-detection function.
+
+---
+
+### Carbon Black Enterprise Protection Pack v1.0.1
+#### Integrations
+##### CarbonBlackProtectionV2  
+Added documentation notes
+
+---
+
+### Carbon Black Enterprise Response Pack v1.0.3
+#### Integrations
+##### _Carbon Black Enterprise Response v2_  
+Fixed an issue where the file context did not behave as expected in the ***cb-get-processes*** command.
+    
+
+#### Playbook
+##### Search Endpoints By Hash - Carbon Black Response V2
+- New playbook - Search Endpoints By Hash - Carbon Black Response V2 playbook with carbonblack-v2.
+
+---
+
+### Check Point Firewall Pack v1.0.2
+#### Integrations
+##### Check Point  
+Deprecated the *ipname* argument from the ***checkpoint-block-ip*** command.
+
+---
+
+### Chronicle Pack v1.1.0
+#### Integrations
+##### Google Chronicle Backstory
+- Added **gcb-list-events** command.
+- Added deep link in existing commands as well as the new command.
+
+---
+
+### Cisco Threat Grid Pack v1.0.2
+#### Integrations
+##### Threat Grid
+- Fixed a bug in **threat-grid-get-analysis-by-id** command which failed on a syntax error.
+
+
+---
+
+### Common Playbooks Pack v1.4.0
+#### Playbooks
+##### Send Investigation Summary Reports  
+Updated the command SearchIncidents to SearchIncidentsV2.
+
+##### Get Original Email - Generic  
+Added an output of email headers.
+
+---
+
+### Common Scripts Pack v1.1.9
+#### Scripts
+##### SetGridField  
+Fixed an issue in which non alphabetically sorted values given to the **columns** were not processed as expected.
+##### VerifyJSON  
+Updated the Docker image to PowerShell 7.
+##### TimeStampCompare  
+Removed an empty tag from the **TimeStampCompare** script.
+##### DateStringToISOFormat  
+Added new transformer script for converting arbitrary date strings to ISO-8601 format.
+
+---
+
+### Common Types Pack v1.2.1
+#### IndicatorTypes
+##### domainRepUnified  
+Updated the ***Domain*** indicator type's default mapping to use the new transformer *DateStringToISOFormat* (where relevant).
+
+---
+
+### Compliance Pack v1.0.1
+#### IncidentFields
+##### New: Secretary Notification
+##### New: Management Notification
+##### New: DPO Notification
+##### New: Media Notification
+##### New: Individuals Notification
+
+---
+
+### Cortex Data Lake Pack v1.0.3
+#### Integrations
+##### Cortex Data Lake
+Adjusted the integration to work with a setup of non-root user in a Docker container.
+
+---
+
+### Cortex XDR - IOC Pack v1.0.1
+#### Integrations
+##### Cortex XDR - IOC  
+Fixed an issue where trying to push a none existed indicator in ***xdr-iocs-push*** raised an error that failed the command.
+
+---
+
+### Deprecated Content Pack v1.2.0
+#### Playbooks
+##### Malware Playbook - Manual
+Deprecated. Use "Malware Investigation - Manual" playbook instead.
+
+---
+
+### EWS Pack v1.1.0
+#### Integrations
+##### EWSO365  
+New Integration EWS O365
```

Comment:
need to improve this
the `New` prefix should be before the integration name and we should have the integration description here instead